service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/documents/ManageDocumentsLAController.java

```diff
@@ -173,6 +178,29 @@ public AboutToStartOrSubmitCallbackResponse handleFurtherEvidenceMidEvent(@Reque
         return respond(caseDetails);
     }
 
+    @PostMapping("/final-check/mid-event")
+    @java.lang.SuppressWarnings("java:S5852")
```

Comment:
Why do we need this?

Reply:
it complains a slow regular expression and I can't get rid of it unless I suppress it. https://sonarsource.atlassian.net/browse/SONARJAVA-3404